### PR TITLE
feat: add CRS base ideas for 2023

### PR DESCRIPTION
**Pull request by fzipi**
First ideas iteration form CRS.

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 1]
```diff
@@ -594,3 +594,162 @@ We'd love to hear it, please reach out via email / owasp slack / github to ensur
...
+- Knowledge about the problem your are trying to solve with this plugin ;)
+
+##### Mentors
+
+The CRS team will assign a mentor to contributors. In the meantime, the following two CRS project leaders will be your contacts:
+
+* [Christian Folini](mailto:christian.folini@owasp.org)
+* [Felipe Zipitría](mailto:felipe.zipitria@owasp.org)
+
+![Difficulty: Easy](https://img.shields.io/badge/difficulty-easy-yellow)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:755]: Green?

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 2]
```diff
@@ -594,3 +594,162 @@ We'd love to hear it, please reach out via email / owasp slack / github to ensur
...
+- Basic knowledge about text conversion functions and Web encodings
+
+##### Mentors
+
+The CRS team will assign a mentor to contributors. In the meantime, the following two CRS project leaders will be your contacts:
+
+* [Christian Folini](mailto:christian.folini@owasp.org)
+* [Felipe Zipitría](mailto:felipe.zipitria@owasp.org)
+
+![Difficulty: Easy](https://img.shields.io/badge/difficulty-easy-yellow)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:643]: Green?

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 1 above)
Comment (fzipi) [reply pages/initiatives/gsoc/gsoc2023ideas.md]: Thanks!

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 2 above)
Comment (fzipi) [reply pages/initiatives/gsoc/gsoc2023ideas.md]: Changed, thanks.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**